chef/cookbooks/crowbar/templates/default/crowbar-shutdown-services-before-upgrade.sh.erb

```diff
@@ -70,10 +70,22 @@ done
 
 <% end %>
 
+# It's possible services are not managed by pacemaker (if clone_stateless_services was not set),
+# so to make sure that services are stopped, use additional systemctl commands.
+log "Stopping OpenStack services..."
+
+for i in /usr/sbin/rcopenstack-* ; do
```

Comment:
cant you just move the `grep -v neutron` to this line so you get only the rest of the services?

Reply:
I guess I could